shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/config/yaml/swapper/YamlProxyDataSourceConfigurationSwapper.java

```diff
@@ -31,9 +33,15 @@
      * Swap YAML proxy data source configuration to data source configuration.
      *
      * @param yamlConfig YAML proxy data source configuration
+     * @param schemaName Schema name
+     * @param dataSourceName Data source name
+     * @param isDataSourceAggregation Is data source aggregation
```

Comment:
`Is` should be `is`

Reply:
Fix in https://github.com/apache/shardingsphere/pull/16711